### PR TITLE
Fix cpu percentage computation

### PR DESCRIPTION
**Pull request by simonepri**
See https://github.com/soyuka/pidusage/issues/58
This should be considered a critical bug-fix rather than a breaking change.

### CODE REVIEW

**Comment by simonepri** [comment]
@soyuka can you re-run the build on appveyor?

File: lib/procfile.js [hunk 1]
```diff
@@ -33,27 +32,28 @@ function readProcFile (pid, options, done) {
...
+    var total = (stat.stime - (hst.stime || 0) + stat.utime - (hst.utime || 0) + childrens) / cpuInfo.clockTick
     // time elapsed between calls in seconds
     var seconds = Math.abs(hst.uptime !== undefined ? stat.uptime - hst.uptime : stat.start - stat.uptime)
-    if (seconds === 0) seconds = 1 // we sure can't divide through 0
+    var cpu = seconds > 0 ? (total / seconds) * 100 : 0
 
     history.set(stat, options.maxage)
 
-    var cpuPercent = Math.min(Math.round((total / seconds) * 100000) / 1000, 100.0)
-    var memory = stat.rss * cpu.pageSize
```
Comment (soyuka) [inline lib/procfile.js]: There's a linting issue actually :)
```
  C:\projects\pidusage\lib\procfile.js:51:1: More than 1 blank line not allowed.
```

I think that appveyor will be happy after!

Comment (simonepri) [reply lib/procfile.js]: Yes I saw it, actually there was a bug in the test command.

**Comment by soyuka** [comment]
Thanks for the patch @simonepri !

**Pull request merged**